Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/branch.py

```diff
@@ -156,7 +168,9 @@ def main(cls, args, repository, why=None, redact=False, target_remote='fork', **
             for reference in issue.references
         ])
 
-        if needs_radar:
+        radar_cc_default = repository.config().get('webkitscmpy.cc-radar', 'true') == 'true'
+        radar_cc_command = getattr(args, 'cc_radar', None)
+        if needs_radar and (radar_cc_command or (radar_cc_default and radar_cc_command is not False)):
```

Comment:
logic nit: If `needs_radar` is true, then (I think?) we know that `args` has a `cc_radar` property. So this can be simplified to:
```suggestion
        if needs_radar and (args.cc_radar or radar_cc_default):
```